### DISCUSSION TITLE
Allow to copy cutomizations from one collection to another

### DISCUSSION THREAD

**Discussion by a-asharapova** [GitbookIO/community#473] [Feature Requests]
Use case: we need several collections with the same looks and feel. That means the same links in the header and footer, and there are a lot of them.

Is it possible to make it so that customizations can be copied from one collection to another?

Thank you!

**Comment by addisonschultz** [comment]
This is definitely a great feature - but you're right, it's not possible at this time. I think it's a good one for us to look into though, so I'll make note of it!
